src/build.py

```diff
@@ -1565,11 +1533,9 @@ def Test(self):
 
 def GetTortureDir(name, opt):
   dirs = {
-      's': os.path.join(TORTURE_S_OUT_DIR, opt),
       'o': os.path.join(TORTURE_O_OUT_DIR, opt),
       'asm2wasm': os.path.join(ASM2WASM_TORTURE_OUT_DIR, opt),
-      'emwasm': os.path.join(EMWASM_TORTURE_OUT_DIR, opt),
-      'emwasm-lld': os.path.join(EMWASM_LLD_TORTURE_OUT_DIR, opt)
+      'emwasm': os.path.join(EMWASM_TORTURE_OUT_DIR, opt)
```

Comment:
Nittiest of nits: can leave the trailing `,` at the end of the line, which reduces line churn in diffs when adding elements, and makes it iotally easier to add or reorder elements.

Not all languages allow this, but python does, and it's pretty nice